templates/terraform/resource_definition/subnetwork.erb

```diff
@@ -1,3 +1,4 @@
 CustomizeDiff: customdiff.All(
 		customdiff.ForceNewIfChange("ip_cidr_range", isShrinkageIpCidr),
+        resourceComputeSubnetworkCustomizeDiff,
```

Comment:
nit: mind making this line tabs or the previous line spaces?

Reply:
Done.